.circleci/config.yml

```diff
@@ -7,8 +7,9 @@ jobs:
       - checkout
       - run: cp .eslintrc ./logdna/.eslintrc
       - run: cd ./logdna && npm install .
-      - run: cd ./logdna && npm install eslint
+      - run: cd ./logdna && npm install eslint jest eslint-plugin-jest
```

Comment:
I think this line is no longer needed here as they have been installed in `npm install .`

Reply:
oops right.